client/components/avatar/UserAvatarEditor.js

```diff
@@ -64,8 +64,8 @@ export function UserAvatarEditor({ username, setAvatarObj, suggestions, disabled
 						<Button square mis='none' onClick={clickReset} disabled={disabled} mie='x4'>
```

Comment:
```suggestion
						<Button square mis='none' onClick={clickReset} disabled={disabled} mie='x4' title={t('Accounts_SetDefaultAvatar')}>
```

Can you add this title too? Thanks!